shell/platform/fuchsia/runtime/dart/utils/BUILD.gn

```diff
@@ -53,10 +56,47 @@ template("make_utils") {
   }
 }
 
+action("generate_build_info_cc_file") {
+  inputs = [
+    "build_info_in.cc",
+    "//flutter/.git/logs/HEAD",
+    "//fuchsia/sdk/linux/meta/manifest.json",
+    "//third_party/dart/.git/logs/HEAD",
+  ]
+  outputs = [ "$target_gen_dir/build_info.cc" ]
+
+  script = "//flutter/tools/fuchsia/make_build_info.py"
+  args = [
+    "--input",
+    rebase_path("build_info_in.cc", root_build_dir),
+    "--output",
+    rebase_path(outputs[0], root_build_dir),
+    "--buildroot",
+    rebase_path("//", root_build_dir),
+  ]
+}
+
 make_utils("utils") {
   deps = [ "//third_party/dart/runtime/bin:elf_loader" ]
 }
 
+executable("dart_utils_unittests") {
+  testonly = true
+
+  output_name = "dart_utils_unittests"
+
+  sources = [ "build_info_unittests.cc" ]
+
+  # This is needed for //third_party/googletest for linking zircon symbols.
+  libs = [ "//fuchsia/sdk/$host_os/arch/$target_cpu/sysroot/lib/libzircon.so" ]
```

Comment:
//fuchsia/sdk/$host_os is $fuchsia_sdk_root which you use below.

Reply:
Good idea! I'll use `$fuchsia_sdk_path` since `fuchsia_sdk_root = "//build/fuchsia"`.